upstairs/src/lib.rs

```diff
@@ -1162,28 +1178,30 @@ where
      * also request extent versions and will have to repair this
      * downstairs, skipping over step 3 as well.
      *
-     * (Offline only):
-     *
+     * NegotiationState::GetLastFlush (offline only)
+     * ------------------------------
      *          Upstairs             Downstairs
-     * 3:       LastFlush(lf)) --->
+     *          LastFlush(lf)) --->
      *                         <---  LastFlushAck(lf)
      *
      * After receiving our last flush, we now move this downstairs state to
      * Replay and skip ahead to step 5.
```

Comment:
Step 5 is now `NegotiationState::Done`

Reply:
Done in 97dfe4778518d3536ecc2adb75bf2b9672296416